src/hotspot/share/jfr/recorder/checkpoint/types/jfrType.cpp

```diff
@@ -214,7 +214,7 @@ void MetaspaceObjectTypeConstant::serialize(JfrCheckpointWriter& writer) {
 static const char* reference_type_to_string(ReferenceType rt) {
```

Comment:
This function seems misplaced here.  Seems like it belongs with the ReferenceType type.  (This could be a followup RFE.)